src/game/client/swarm/vgui/campaignpanel.cpp

```diff
@@ -1040,6 +1044,13 @@ void CampaignPanel::OnCommand(const char *command)
 		}
 #endif
 	}
+	else if ( !Q_stricmp( command, "ChangeMissionButton" ) )
+	{
+		if ( ASWGameRules() && ASWGameRules()->GetGameState() == ASW_GS_INGAME )
+			engine->ClientCmd( "asw_vote_chooser 0" );
+		else
+			engine->ClientCmd( "asw_vote_chooser 0 notrans" );
```

Comment:
the command is `asw_mission_chooser callvote`

Reply:
I updated the command.